webpack.config.js

```diff
@@ -11,22 +11,19 @@ const validEnvironments = [
   'staging',
   'production',
 ]
-const environment = process.env.NODE_ENV
+const environment = process.env.NODE_ENV || 'development'
 
 assert(validEnvironments.includes(environment), `Invalid value for NODE_ENV: ${environment}. Valid values are: ${validEnvironments}`)
 
 const production = environment === 'production'
 const development = environment === 'development'
 const configurationPath = `./env/${environment}.json`
-const redirects = `./_redirects.${environment}`
```

Comment:
I think to fix the redirect issue for Netlify this line(and probably should include the console.log below) need to be added back in.